content/applications/finance/accounting/reporting/year_end.rst

```diff
@@ -76,40 +79,76 @@ Next, the accountant likely verifies balance sheet items and book entries for:
   - tax adjustments,
   - etc.
 
-If the accountant is going through the year-end audit, they may want to have paper copies of all
-balance sheet items (such as loans, bank accounts, prepayments, sales tax statements, etc.) to
-compare these with the balances in Odoo.
+During the year-end audit, the accountant may print paper copies of all balance sheet items (e.g.,
+loans, bank accounts, prepayments, sales tax statements) to compare them against the balances
+recorded in Odoo.
 
 .. tip::
-   During this process, it is good practice to set a :guilabel:`Journal Entries Lock Date` to the
-   last day (inclusive) of the preceding fiscal year by going to :menuselection:`Accounting -->
-   Accounting --> Lock Dates`. This way, the accountant can be confident that nobody changes the
-   transactions while auditing the books. Users from the *accountant* access group can still create
-   and modify entries.
+   As part of this process, setting a :ref:`Lock Everything <year-end/lock-everything-date>` date to
+   the last day (inclusive) of the preceding fiscal year is good practice. This ensures that journal
+   entries with an accounting date on or before the lock date cannot be created or modified during
+   the audit. Users with *administrator* access rights can still create and edit entries if an
+   exception is configured.
+
+.. _year-end/lock-everything-date:
+
+Lock everything date
+~~~~~~~~~~~~~~~~~~~~
+
+Setting a lock date prevents modifications to any posted journal entries with an accounting date on
+or before the lock date. It also prevents posting new entries with an accounting date on or before
+the lock date. In such cases, the system automatically sets the accounting date to the day after the
+lock date.
+
+To set a :guilabel:`Lock Everything` date, go to :menuselection:`Accounting --> Accounting --> Lock
+Dates`. In the :guilabel:`Lock Journal Entries` window, set the :guilabel:`Lock Everything` date and
+:guilabel:`Save`.
+
+.. note::
+   Users with :guilabel:`Administrator` access rights to the Accounting app can create exceptions.
+   To do so:
+
+   #. After setting the :guilabel:`Lock Everything` date, reopen the :guilabel:`Lock Journal
+      Entries` window and remove the :guilabel:`Lock Everything` date.
+   #. In the :guilabel:`Exception` banner, choose if this exception should be set :guilabel:`for me`
+      (the current user) or :guilabel:`for everyone` and how long it should last.
+   #. A :guilabel:`Reason` for this exception can be added.
+   #. All of this information is logged in the chatter of the :doc:`company record
+      </applications/general/companies>`.
+
+.. note::
```

Comment:
```suggestion
.. tip::
```
I would make this a tip or merge the two notes together using bullet points, like this:

- Note 1

  #. (...)
  #. (...)

- Note 2